funding/manager.go

```diff
@@ -3246,10 +3254,13 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 	shutdown, err := getUpfrontShutdownScript(
 		f.cfg.EnableUpfrontShutdown, msg.Peer,
 		msg.ShutdownScript,
-		func() (lnwire.DeliveryAddress, error) {
+		func(taprootOK bool) (lnwire.DeliveryAddress, error) {
```

Comment:
nit: dedupe?

Reply:
Good call, made a helper func to resolve it